components/home/cards.html

```diff
@@ -7,8 +7,10 @@ <h5 class="card-title">Fitness, Fellowship and Faith</h5>
                     F3 is a national network of free, peer-led workouts for men of all ages and fitness levels. Our mission is to plant, grow and serve small workout groups for men for the invigoration of male community leadership.
                 </p>
             </div>
-            <div class="card-footer">
-                <a href="#" class="btn btn-primary">About F3</a>
+            <div class="card-footer d-flex justify-content-center">
```

Comment:
Button alignment - I'd prefer that we keep the buttons left justified.

My rational is that when scanning the text box left to right reading through the card, after getting to the end of the last sentence your eye is going back to that left position to read the next line.  Being that we are updating the links to be to be actions like `Find a workout` or `Learn more about F3` then to me it makes sense to align them to the left as they are part of the card text, but seperated because its a call to action.

If you feel strongly about centering the calls to action, lets add a css rule for .card-footer to justify center instead of updating each one

Reply:
I tried both left and center. Think it looked more pleasing centered, but not married to that.